src/_includes/feature_lists/self-hosted.njk

```diff
@@ -0,0 +1,106 @@
+{%
+    set sections = [{
+        label: "Features",
+        hosting: "self-hosted",
+        rows: [{
+            id: "team-members",
+            label: "Team Members",
+            values: ['2', '20', 'Unlimited'],
+            info: "<p>FlowForge allows multiple team members to collaborate on the same Node-RED flows.</p>"
+        }, {
+            id: "cloud-instances",
+            label: "Number of cloud instances",
+            values: ['2', 'Licensed amount', 'Licensed amount'],
+            info: "<p>FlowForge hosts multiple Node-RED instances from a common platform.</p>"
+        }, {
+            id: "devices",
+            label: "Number of devices",
+            values: ['2', 'Licensed amount', 'Licensed amount'],
+            info: "<p>FlowForge allows you to manage Node-RED deployments on remote devices.</p>"
+        }, {
+            id: "nr-hosting",
+            label: "Node-RED Hosting",
+            values: ['check', 'check', 'check'],
+            info: "<p>FlowForge hosts Node-RED instances.</p>"
+        }, {
+            id: "audit-log",
+            label: "Audit Log",
+            values: ['check', 'check', 'check'],
+            info: "<p>Keep track of everything going on in your Node-RED instances and FlowForge.</p><p>Audit Logs provide details on what actions have taken place, when they happened, and who did them.</p>"
+        }, {
+            id: "pipelines",
+            label: "DevOps Pipelines",
+            values: ['check', 'check', 'check'],
+            info: "<p>DevOps pipelines allows for you to setup different environments for development, testing and production Node-RED instances.</p>"
+        }, {
+            id: "device-mgmt",
+            label: "Device Management",
+            values: ['check', 'check', 'check'],
+            info: "<p>Connect to edge devices to quickly assess and update logic. Debug one device and roll out improvements to your fleet in minutes. All securely without requiring full device access for your whole organisation.</p>"
+        }, { 
+            id: "rbac",
+            label: "Role-Based Access Control",
+            values: ['check', 'check', 'check'],
+            info: "<p>FlowForge's intuitive team management tooling makes it easy for you to control whom has access to what.</p>"
+        }, {
+            id: "security",
+            label: "Endpoint Security",
+            values: ['None/HTTP Basic Auth', 'None/HTTP Basic Auth/FF Auth', 'None/HTTP Basic Auth/FF Auth'],
```

Comment:
```suggestion
            values: ['None/HTTP Basic Auth/FF Auth', 'None/HTTP Basic Auth/FF Auth', 'None/HTTP Basic Auth/FF Auth'],
```

Reply:
Consider relabeling the row to SSO Endpoint Security, with enterprise having a ✔️

Reply:
Did we didn't discuss this in the meeting? What should I do with this one?

Reply:
Thanks @MarianRaphael